checker/src/main/java/org/checkerframework/checker/initialization/InitializationVisitor.java

```diff
@@ -266,8 +186,8 @@ public Void visitTypeCast(TypeCastTree tree, Void p) {
             checker.reportError(
                     tree,
                     COMMITMENT_INVALID_CAST,
-                    annoFormatter.formatAnnotationMirror(exprAnno),
-                    annoFormatter.formatAnnotationMirror(castAnno));
+                    atypeFactory.getAnnotationFormatter().formatAnnotationMirror(exprAnno),
```

Comment:
We could also make the field https://github.com/eisop/checker-framework/blob/6376cb64ed59d9f3319cad206fa4de619b92f823/framework/src/main/java/org/checkerframework/framework/type/AnnotatedTypeFactory.java#L276 protected.
The neighboring `typeFormatter` is protected, so why not the annotation formatter, too?